src/LearnositySdk/Request/Init.php

```diff
@@ -17,6 +17,15 @@
 
 class Init
 {
+    const VERSION_FILE_PATH = __DIR__ . '/../../../.version';
```

Comment:
@shtrom done here and below - check last commit in the stack: https://github.com/Learnosity/learnosity-sdk-php/pull/33/commits/8477692b1804a38b9c50854f8224b619ba43aed4